apps/docs-snippets2/src/wallet/getBalance.ts

```diff
@@ -0,0 +1,9 @@
+import { fuels, TESTNET_NETWORK_URL } from 'fuels/api';
+
+const MY_WALLET_ADDRESS = '0x767caf5b08eba21c561078a4d5be09bbd7f16b9eca22699a61f1edd9e456126f';
+
+const { wallet } = await fuels(TESTNET_NETWORK_URL);
+
+const balances = await wallet(MY_WALLET_ADDRESS).getBalances();
+
+console.log({ balances });
```

Comment:
How to use `launchTestNode` without polluting the code snippet?

Reply:
The Stackblitz topic runs [independently](https://github.com/FuelLabs/fuels-ts/issues/1927#issuecomment-2198814457) of this one, but at one point, I think we'd want that.

Reply:
cc @danielbate @nedsalk